every_error_text.js

```diff
@@ -0,0 +1,13 @@
+export const everyErrorText = {
+   
+	THE_NAME_MUST_CONTAIN_AT_LEAST_3_LETTERS:
+		"The name must contain at least 3 letters",
+	FORBIDDEN_SPECIAL_CHARACTERS: `Forbidden special characters`,
+	PASSWORD_IS_TOO_SHORT: "Password is too short, minimum 8 characters",
+	SPECIAL_CHARACTERS: `Use special character`,
```

Comment:
s missing on the end of the line?

Reply:
no, because the code expects minimum one character